test/examples_test.go

```diff
@@ -0,0 +1,35 @@
+package test
+
+import (
+	"path/filepath"
+	"testing"
+)
+
+const examplesDir = ".."
+const scriptDir = "scripts"
+
+func TestCoinPricesFromCoingecko(t *testing.T) {
+	projectName := "coin_prices_from_coingecko"
+	projectDir := filepath.Join(examplesDir, projectName)
+	requiredEnvVars := []string{
+		"YOUR_COINGECKO_API_KEY",
+	}
+
+	NewProjectTest(t, projectDir).
+		ExecuteMakeTarget("build").
+		CopyFile("tmp/x.wasm").
+		CopyFile("config.toml").
+		RenderTemplateFileFromEnvWithCleanup("fn-call.json", requiredEnvVars).
+		RunScript(filepath.Join(scriptDir, projectName+".txtar"))
```

Comment:
This setup expects a single script. This uses the `testscript.Params.Files` field under the hood. We could easily update this to run multiple scripts in the same testscript WorkDir if we wanted to. I did not see a big reason for this at the moment, so I left it as a "one shot"